roadmap/implementors-guide/src/node/utility/candidate-validation.md

```diff
@@ -2,12 +2,24 @@
 
 This subsystem is responsible for handling candidate validation requests. It is a simple request/response server.
 
+A variety of subsystems want to know if a parachain block candidate is valid. None of them care about the detailed mechanics of how a candidate gets validated, just the results. This subsystem handles those details.
+
 ## Protocol
 
-Input:
+Input: [`CandidateValidationMessage`](/type-definitions.html#validation-request-type)
 
-- [`CandidateValidationMessage`](/type-definitions.html#validation-request-type)
+Output: [`Statement`](/type-definitions.html#statement-type) via the provided `Sender<Statement>`.
 
 ## Functionality
 
-Given a candidate, its validation code, and its PoV, determine whether the candidate is valid. There are a few different situations this code will be called in, and this will lead to variance in where the parameters originate. Determining the parameters is beyond the scope of this subsystem.
+Given the hashes of a relay parent and a parachain candidate block, and either its PoV or the information with which to retrieve the PoV from the network, spawn a short-lived async job to determine whether the candidate is valid.
```

Comment:
The old line said: 

> Determining the parameters is beyond the scope of this subsystem.

 I find this vastly superior to a system where the validation subsystem is responsible for fetching the PoV. The caller will know best where to get the PoV.

Reply:
I didn't know what "determining the parameters" meant, in the old line. 

If the caller is responsible for fetching the PoV, then what is this subsystem supposed to do when it receives a `CandidateValidationMessage::Validate` with `PoVOrigin::Network`? If it's the caller's responsibility, then we don't need `PoVOrigin` at all; we can just make the PoV one of `Validate`'s fields.

Reply:
> If it's the caller's responsibility, then we don't need PoVOrigin at all; we can just make the PoV one of Validate's fields

Yeah, this is what I'd advocate for.

Reply:
OK, I can make that change.